delivery_roulier_laposte/models/stock_package.py

```diff
@@ -28,6 +28,10 @@
 class StockQuantPackage(models.Model):
     _inherit = 'stock.quant.package'
 
+    # you customize the format overriding _get_label_format()
+    # required to know what is the format after call
```

Comment:
we can return the requested format with the call
https://github.com/akretion/roulier/wiki/Harmonize-response-type